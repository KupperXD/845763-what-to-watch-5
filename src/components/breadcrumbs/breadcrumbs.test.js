import React from 'react';
import renderer from 'react-test-renderer';
import {BrowserRouter} from "react-router-dom";

import Breadcrumbs from './breadcrumbs';

const film = {};

it(`renders correctly`, () => {
  const bread = renderer
    .create(<BrowserRouter>
      <Breadcrumbs film={film} />
    </BrowserRouter>).toJSON();

  expect(bread).toMatchSnapshot();
});
