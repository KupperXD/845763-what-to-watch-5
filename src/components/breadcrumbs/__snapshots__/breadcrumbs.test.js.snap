// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="breadcrumbs"
>
  <ul
    className="breadcrumbs__list"
  >
    <li
      className="breadcrumbs__item"
    >
      <a
        className="breadcrumbs__link"
        href="/films/undefined"
        onClick={[Function]}
      />
    </li>
    <li
      className="breadcrumbs__item"
    >
      <a
        className="breadcrumbs__link"
      >
        Add review
      </a>
    </li>
  </ul>
</nav>
`;
